wordpress-mobile/gutenberg-mobile
# Update Appium dependency version to 1.20.2

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/28890

## Description
Appium dependency has recently included a fix for preventing an error when using NPM version 7 so it would be nice to update it.

To test:
Appium is only used for running the end-to-end tests so we need to verify that they work as expected. It should be enough by verifying that the PR checks pass and by running them locally with the following commands:
1. `npm run test:e2e:ios:local` for iOS
2. `npm run test:e2e:android:local` for Android

PR submission checklist:
